app/models/bike.rb

```diff
@@ -64,7 +64,11 @@ class Bike < ActiveRecord::Base
 
   attr_writer :phone, :user_name # reading is managed by a method
 
-  default_scope { where(example: false).where(hidden: false).order('listing_order desc') }
+  default_scope { 
```

Comment:
Why add these to the default scope?

Reply:
There's a slew of n+1's without these eager loaded.  Seems like the `default_scope` makes the most sense to avoid duplication;  if not you can just alter the respective queries that kick off from `/bikes` w/ the same includes